app/generated_resources_override.grd

```diff
@@ -331,6 +331,15 @@ are declared in tools/grit/grit_rule.gni.
         </message>
         </if>
 
+      <if expr="is_macosx">
+            </if>
+      <if expr="not is_macosx">
+        <if expr="not use_titlecase">
+            </if>
+          <if expr="use_titlecase">
+            </if>
+      </if>
```

Comment:
why are we adding these empty blocks? is this left because of some script?

Reply:
Yes, those are from `brave/build/commands/lib/chromiumRebaseL10n.js` (`brave/script/chromium-rebase-l10n.py`).